### PR TITLE
Add GH Actions for running eslint, unit and integration tests

### PR DESCRIPTION
**Pull request by daynin**

### CODE REVIEW

**Comment by stchang** [comment]
Thanks for starting to set these up. Let us know if we can help, or answer any questions. I'm not fluent with gh actions yet, but does the integration test workflow need to install js?

**Comment by daynin** [comment]
@stchang thank you! But I've finished just now

> I'm not fluent with gh actions yet, but does the integration test workflow need to install js?

No, there will be two separate actions (for elsint and for racket tests) and they will install their deps independently.

Btw, looks like tests pass almost for every racket version (excluding 6.9 and 8.1).

File: racketscript-compiler/info.rkt [hunk 1]
```diff
@@ -19,6 +19,7 @@
 
 (define pkg-authors '(vishesh))
 (define pkg-desc "Racket to JavaScript compiler")
+(define post-install-collection "")
```
Comment (daynin) [inline racketscript-compiler/info.rkt:22]: I don't know why but without that variable installation failed in GH Actions

**Comment by stchang** [comment]
Ah great. Let me know when this is ready to merge

File: racketscript-compiler/info.rkt (hunk 1 above)
Comment (stchang) [reply racketscript-compiler/info.rkt:22]: Weird, I havent seen this before. Thanks for figuring it out.

**Comment by daynin** [comment]
@stchang looks like, now it's ready

**Comment by daynin** [comment]
But pay an attention please, I deleted the TravisCI config

**Comment by stchang** [comment]
> But pay an attention please, I deleted the TravisCI config

Yes, that's good. We wanted to get rid of it anyways

**Comment by vishesh** [comment]
This is awesome! Thanks!!

**Pull request merged**